Source/SwiftLintFramework/Extensions/File+Cache.swift

```diff
@@ -68,7 +68,7 @@ private struct Cache<T> {
 extension File {
 
     private var cacheKey: String {
-        return path ?? "\(ObjectIdentifier(self).hashValue)"
+        return path ?? "\((contents as NSString).hash)"
```

Comment:
On most of our usage, `path` will be used. I think the memory overhead can be ignored.